source/_posts/Integral-Image.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Link to previous content - in content there should be link with previous content

Reply:
done

Reply:
Done